src/Input.js

```diff
@@ -0,0 +1,76 @@
+import { React, Component } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import './Input.css'
+
+class Input extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchInput: '', 
+      rating: ''
+    }
+  }
+
+  setInput = (e) => {
+    this.setState({ searchInput: e.target.value })
+  }
+
+  setRating = (e) => {
+    this.setState({ rating: e.target.value })
+  }
+
+  findMovie = (e) => {
+    e.preventDefault();
+    if(this.state.searchInput) {
+      const foundMovie = this.props.movies.find((movie) => {
+        return movie.title === this.state.searchInput || movie.title === this.state.searchInput.toLowerCase() || movie.title === this.state.searchInput.toUpperCase();
```

Comment:
Would it be possible to set both the movie.title and search input to lower case so that they match regardless of how the user types them?